portal/app/algorithms/battery_load_control.py

```diff
@@ -0,0 +1,52 @@
+import json, datetime
+import numpy as np
+from app.farm_updater import sonnen_api
+
+
+def batt_dispatch():
+    from app.models import FarmDevice, FarmData, DeviceType
+
+    egauge_device = FarmDevice.objects.get(device_uid='46613')
+    sonnen_queryset = FarmDevice.objects.filter(type=DeviceType.SONNEN)
+    farmdata = FarmData.objects.filter(farm_device=egauge_device).order_by('-id')[0:3]
+    test_pen_power = []
+
+    # Getting egauge info
+    for data in farmdata:
+        test_pen_power.append(json.loads(data.device_data)["processed"]["POWER_CIRCUIT1"]+json.loads(data.device_data)["processed"]["POWER_CIRCUIT2"])
```

Comment:
Settle on one style of quotes. Let's just use single quotes everywhere. Also, put whitespaces around `+`